bin/setup

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+require 'pathname'
+require 'optparse'
+
+
+options = {:do_tests => true, :do_db => true}
+OptionParser.new do |opts|
+  opts.banner = "Usage: setup [options]"
+  opts.on("-d", "--no-db", "Do not prepare db") do
+    options[:do_db] = false
+  end
+  opts.on("-t", "--no-tests", "Do not perform tests") do
+    options[:do_tests] = false
+  end
+  opts.on("-h", "--help", "Displays this help") do
+    puts opts
+    exit
+  end
+end.parse!
+
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+
+Dir.chdir APP_ROOT do
+  puts "== Installing dependencies =="
+  system "gem install bundler --conservative"
+  system "bundle check || bundle install"
+
+  puts "\n== Copying sample files =="
+  unless File.exist?("config/database.yml")
+    system "cp config/database.tmpl.yml config/database.yml"
+  end
+  unless File.exist?("Procfile")
+    system "cp Procfile.tmpl Procfile"
+  end
+  unless File.exist?("config/settings/development.local.yml")
+    system "cp config/settings.yml config/settings/development.local.yml"
```

Comment:
I don't think it makes sense to copy `config/settings.yml`.  I would only expect that file to contain intentional differences.